velox/substrait/SubstraitToVeloxPlan.cpp

```diff
@@ -705,4 +605,33 @@ const std::string& SubstraitVeloxPlanConverter::findFunction(
   return substraitParser_->findFunctionSpec(functionMap_, id);
 }
 
+void SubstraitVeloxPlanConverter::setPhase(
```

Comment:
- Change the signature to return core::AggregationNode::Step instead of taking it as an out parameter.
- Rename to toAggregationStep.
- Make this a standalone function in an anonymous namespace.